protocol/contracts/beanstalk/metadata/MetadataFacet.sol

```diff
@@ -19,6 +21,10 @@ import "./MetadataImage.sol";
  */
 contract MetadataFacet is MetadataImage {
     using LibStrings for uint256;
+    using LibStrings for int256;
+
+
```

Comment:
Can you remove one of these blank lines?

Reply:
fixed: c58be9b7fa3eda06805602e40e1e93f0fb281137